docs/guidance/model/organization_classifications.md

```diff
@@ -0,0 +1,86 @@
+
+```eval_rst
+
+.. jsoninclude:: ../../examples/organization-classification/uk_organization_classification.json
+   :jsonpointer: /releases/0/parties/0
+   :expand: details, classifications
+   :title: organization_classifications
+
+```
+
+#### Example 2.2 disclosing data using a local scheme
+The second example below is set in the fictional city of Ciudad Ficticia in Colombia. The procurement team wishes to monitor the participation of women-owned businesses in procurement according to specific policy priorities. The first organization declared in the `parties` array is a women-owned business, so they add a `classifications` array to it with just one `classification` object. In this object, the local  `classification.scheme` is 'COL-CF-MON' and the `classification.id` is 'NPDM'. Note that the `classification.id` contains the relevant classification code from the given scheme, rather than being an internal identifier in the `classifications` array.
+
+In their publication policy, the procurement team documents all possible codes for COL-CF-MON with definitions of each code, including explaining that 'NPDM' is for businesses registered with the local Chamber of Commerce where ownership and control is at least 51% women.
+
+```eval_rst
+
+.. jsoninclude:: ../../examples/organization-classification/fictional_wob_organization_classification.json
+   :jsonpointer: /releases/0/parties/0
+   :expand: details, classifications
+   :title: organization_classification
+
+```
+
+### Option 3: Local extensions
+
+A third, not-recommended option is for publishers to use local extensions to disclose organization classification information. This option is not recommended because it is difficult for data users to compare organization classifications across multiple data sets that use many different approaches to disclosing similar information. However, in the absence of standardized options, where there is a specific use case for the data, this may be the most appropriate short-term option. Local extensions should document the structure and meaning of the additional fields they describe: please refer to the [extensions documentation](extensions).
+
+For example, although tracking women-owned organizations is shown example 2.2 above, this data only provides information on entities that have been registered as women-owned. Organizations without the classification may be not women-owned, women-owned but not registered as such, or the information may not be known.
+
+To disambiguate these cases, a publisher may choose to publish a flag field for the relevant organization classification. In the fictional example below, Dhanghadi has created a local extension so they can publish data in the `parties.details` block on an organization that is `femaleChaired`, with the values of the field being either 'yes' or 'no'. The publisher would document the structure of this field and its meaning in the local extension files.
```

Comment:
The example uses `true` as a value for `femaleChaired`, should we change the values of the field to "either true or false"?